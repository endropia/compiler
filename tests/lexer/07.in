'This is a pascal string'
''
'a'
'a'#66#67#68
'A tabulator character: '#67' is easy to embed'
'the string starts here
   and continues here'