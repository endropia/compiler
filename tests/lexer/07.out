1	1	String	This is a pascal string	'This is a pascal string'
2	1	String		''
3	1	String	a	'a'
4	1	String	aBCD	'a'#66#67#68
5	1	String	A tabulator character: C is easy to embed	'A tabulator character: '#67' is easy to embed'
(6, 24) Unterminated string